### PR TITLE
docs(controller): use async

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: docs/source/en/basics/controller.md [hunk 1]
```diff
@@ -700,44 +705,55 @@ Most data is sent to receivers through the body and, just like the body in the r
...
 #### Rendering Template
 
 Usually we do not write HTML pages by hand, instead we generate them by a template engine.
 Egg itself does not integrate any template engine, but it establishes the [View Plugin Specification](../advanced/view-plugin.md). Once the template engine is loaded, `ctx.render(template)` can be used to render templates to HTML directly.
+
+```js
+class HomeController extends Controller {
+  async index() {
+    const ctx = this.ctx;
+    ctx.render('home.tpl', { name: 'egg' });
```
Comment (dead-horse) [inline docs/source/en/basics/controller.md]: await ctx.render ?

File: docs/source/zh-cn/basics/controller.md [hunk 2]
```diff
@@ -751,7 +743,18 @@ module.exports = class ProxyController extends Controller {
...
 
 通常来说，我们不会手写 HTML 页面，而是会通过模板引擎进行生成。
-Egg 自身没有集成任何一个模板引擎，但是约定了 [View 插件的规范](../advanced/view-plugin.md)，通过接入的模板引擎，可以直接使用 `ctx.render(template)` 来渲染模板生成 html。
+框架自身没有集成任何一个模板引擎，但是约定了 [View 插件的规范](../advanced/view-plugin.md)，通过接入的模板引擎，可以直接使用 `ctx.render(template)` 来渲染模板生成 html。
+
+```js
+class HomeController extends Controller {
+  async index() {
+    const ctx = this.ctx;
+    ctx.render('home.tpl', { name: 'egg' });
```
Comment (dead-horse) [inline docs/source/zh-cn/basics/controller.md]: await

Comment (atian25) [reply docs/source/zh-cn/basics/controller.md]: fixed

**Comment by dead-horse** [comment]
@atian25 下次不需要本地 rebase 了，反倒看不了 fix 了什么，翻半天，合并的时候 squash 就好

**Comment by atian25** [comment]
好

**Pull request merged**